mock/world/worldUpdate.go

```diff
@@ -58,14 +58,16 @@ func (b *MockWorld) UpdateAccounts(
 		acct := b.AcctMap.GetAccount(modAcct.Address)
 		if acct == nil {
 			acct = &Account{
```

Comment:
You can use `acct = b.AcctMap.CreateAccount(modAcct.Address)` and then set `acct.OwnerAddress = modAcct.CodeDeployerAddress`.

If needed, modify `b.AcctMap.CreatAccount()` to initialize the extra fields.